src/test/integration/controllers/auth.spec.js

```
@@ -114,4 +114,142 @@
       expectError(400, errors.BAD_RESET_TOKEN, response)
     })
   })
+
+  describe('Google Auth endpoint', () => {
+    const mockGooglePayload = {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)